lldb/test/API/lang/swift/async/expr/main.swift

```diff
@@ -0,0 +1,14 @@
+actor Actor {
+  func f(_ n : Int) async {
+    f(0)     // Force a coro-split.
```

Comment:
Does this not require an `await`?

Why does this require a forced coroutine split?

Reply:
It doesn't. Originally I thought a .resume function would be more interesting, but there isn't really a difference for this case. I'll remove it.